packages/shared.net.platformapi/src/foundryPlatformFetch.ts

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
```

Comment:
This is what's on main, so doesn't have my URL parsing changes but _does_ have Nihal's response blob change.